core/src/migrations/sqlite/1609105701370-AddPronunciationField.ts

```diff
@@ -0,0 +1,168 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class AddPronunciationField1609105701370 implements MigrationInterface {
+    name = 'AddPronunciationField1609105701370'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`DROP INDEX "IDX_763d9599de97b88c1adcd12dac"`);
```

Comment:
sqlite can add columns without recreating the whole database. typeorm seems to have a default (and dumb) process in place here. compare other sqlite-migrations for how to do this.

Reply:
i think this needs to be refactored manually. as said above, previous migrations can be taken as a template. otherwise user devices will lock up and probably crash.

Reply:
done